packages/lesswrong/lib/collections/messages/schema.js

```diff
@@ -39,10 +39,16 @@ const schema = {
     viewableBy: ['members'],
     insertableBy: ['members'],
     editableBy: Users.owns,
-    control: 'CommentEditor',
+    control: 'EditorFormComponent',
```

Comment:
I'm confused about why the messages schema still has the content field, whereas the comments schema seems not to. (I suppose this is just a function-preserving PR, and the messages will get updated in the next pass?)

Reply:
Ah, nope. That should actually be killed. Thanks for noticing.